test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -502,68 +502,80 @@ def forward(self, x):
+            __constants__ = ['mode', 'use_size', 'is_upsample', 'size', 'scale', 'size_array', 'scale_array', 'align_corners']
 
-            def __init__(self, mode, use_size, is_upsample):
+            def __init__(self, mode, use_size, is_upsample, align_corners):
                 super(MyModel, self).__init__()
                 self.mode = mode
                 self.use_size = use_size
                 self.is_upsample = is_upsample
+                self.align_corners = align_corners
                 self.scale = 2.0 if self.is_upsample else 0.5
                 self.size = 24 if self.is_upsample else 1
                 if x.dim() == 3:
-                    self.scale_array = [2.]
+                    self.scale_array = [2.3]
                     self.size_array = [16]
                 elif x.dim() == 4:
-                    self.scale_array = [2., 3.]
+                    self.scale_array = [2.3, 3.1]
                     self.size_array = [16, 32]
                 else:
-                    self.scale_array = [2., 3., 4.]
+                    self.scale_array = [2.3, 3.1, 4.6]
                     self.size_array = [16, 32, 64]
 
             @torch.jit.script_method
             def forward(self, x):
                 if self.use_size:
-                    out = torch.nn.functional.interpolate(x, mode=self.mode, size=self.size)
-                    out_array = torch.nn.functional.interpolate(x, mode=self.mode, size=self.size_array)
-                    return out, out_array
-                out = torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale)
-                out_array = torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale_array)
-                return out, out_array
-
-        model = MyModel(mode, use_size, is_upsample)
+                    if self.align_corners:
+                        return torch.nn.functional.interpolate(x, mode=self.mode, size=self.size, align_corners=True), \
+                            torch.nn.functional.interpolate(x, mode=self.mode, size=self.size_array, align_corners=True)
+                    return torch.nn.functional.interpolate(x, mode=self.mode, size=self.size), \
+                        torch.nn.functional.interpolate(x, mode=self.mode, size=self.size_array)
+                if self.align_corners:
+                    return torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale), \
+                        torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale_array)
+                return torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale), \
+                    torch.nn.functional.interpolate(x, mode=self.mode, scale_factor=self.scale_array)
+
+        model = MyModel(mode, use_size, is_upsample, align_corners)
         self.run_test(model, x)
 
     # TODO: Enable bicubic, linear1d and linear3d when implemented in ORT
```

Comment:
Are these tests enabled now?

Reply:
yes! cubic is the only one that is still disabled